support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -112,8 +112,8 @@ function getIsRemoteFromAcquisitionData(): boolean {
 	}
 
 	try {
-		const data = JSON.parse(queryString);
-		return data && !!data.isRemote;
+		const data = JSON.parse(queryString) as Record<string, unknown>;
```

Comment:
A bit unsure on how to type this but open to suggestions!

Reply:
How about `const data = JSON.parse(queryString) as { isRemote?: boolean };` , this is more explicit about the expected return type.

Reply:
[here's](https://github.com/guardian/support-dotcom-components/blob/main/packages/shared/src/lib/tracking.ts#L34) the model in sdc that is added to the query params. We could try and match this type, but it might be easiest just to leave it like this!